src/authoring/GUI.java

```diff
@@ -47,6 +47,7 @@
  */
 public class GUI extends JFrame {
 	private static final long serialVersionUID = -1291725446662111704L;
+	private static final int fileSizeLimit = 1024000; //1MB file size limit for log file
```

Comment:
Maybe we could pump the limit to 100mb, or only enable logging while on debug mode. Right now logging will be generated regardless if we are debugging the app or not, that means the production version of it (the one being used in the wild) will be generating logs all day. The app would end up breaking after some usage

Reply:
The logger class now has a file limit of 100mb per log file.